cfme/infrastructure/config_management.py

```diff
@@ -401,6 +404,75 @@ def delete(self, cancel=False, wait_deleted=True, force=False):
                 # check the provider is indeed deleted
                 assert not self.exists
 
+    @property
+    def rest_api_entity(self):
+        """ Returns the rest entity of config manager provider
+        """
+        # Since config manager provider is slightly different from other normal providers,
+        # we cannot obtain it's rest entity the normal way, instead, we use Entity class
+        # of manageiq_client library to instantiate the rest entity.
+        provider_id = self.appliance.rest_api.collections.providers.get(
+            name=self.ui_name
+        ).provider_id
+
+        return RestEntity(
+            self.appliance.rest_api.collections.providers,
+            data={
+                "href": self.appliance.url_path(
+                    "/api/providers/{}?provider_class=provider".format(provider_id)
+                )
+            },
+        )
+
+    def create_rest(self):
+        """Create the config manager in CFME using REST"""
+        if "ansible_tower" in self.key:
```

Comment:
Q: is there any better way than this check?... suspect about `name` or `key` they can be different as per me.

Reply:
I checked different properties of the config manager and `key` seemed like the best bet. But we could use `config_manager.yaml_data['type']` if not this.